src/components/Speciality/Projects.js

```diff
@@ -4,38 +4,60 @@ import { Row, Col, Grid } from '../grid'
 import { SectionTitle, CardTitle, Subtitle, BodyPrimary } from '../Typography'
 import { Padding } from 'styled-components-spacing'
 import breakpoint from 'styled-components-breakpoint'
+import getColorLuminance from '../../utils/getColorLuminance'
 import { AnimatedLink, CardHeader, PosterImage } from '../Common/animatedLink'
 import LogoGrid from '../Common/LogoGrid'
 
 const Emphasis = styled.em`
   color: ${props => props.theme.colors.secondaryText};
 `
 
-const PosterLinks = ({ project }) => (
-  <AnimatedLink to={`/case-study/${project.slug}`} title={project.title}>
-    <section
-      style={{
-        background: `#${project.posterColor}`
-      }}
-    >
-      <CardHeader>
-        <CardTitle reverse noPadding bigger>
-          {project.title}
-        </CardTitle>
-        <BodyPrimary reverse muted>
-          {project.introSentence.introSentence}
-        </BodyPrimary>
-      </CardHeader>
-      <PosterImage justifyCenter alignCenter color={project.posterColor}>
-        <img
-          alt={project.posterImage.title}
-          src={project.posterImage.file.url}
-          style={{ maxHeight: '100%' }}
-        />
-      </PosterImage>
-    </section>
-  </AnimatedLink>
-)
+const PosterLinks = ({ project }) => {
+  const preferAdditionalPoster =
+    project.slug === 'central-working-changing-how-you-work'
+
```

Comment:
To allow us to not have to update this code each time we want to add an alternative poster image I would check for the existence of additionalPoster in the project object instead.

```
const preferAdditionalPoster = project && project. additionalPoster ! == null
```

Reply:
If you wanted to be really picky you could do: 

```
const imageKey = project && project. additionalPoster ! == null ? 'additionalPoster' : 'posterImage'

const poster = {
    title: project[imageKey].title
    url: project[imageKey].file.url,
    color: project.posterColor
  }
```

Reply:
But we don't know if we will always prefer the additional poster for the Projects, additional poster might not be null in the cases we don't want to use it here.

Reply:
But we will only ever put the image in the field in the CMS if we want it right?